packages/slice-machine/package.json

```diff
@@ -38,9 +38,8 @@
   },
   "dependencies": {
     "@babel/register": "^7.17.7",
-    "@prismicio/slice-simulator-com": "^0.2.0",
```

Comment:
did we remove slice-simulator?

Reply:
it's in the devDependencies :)

Reply:
What about for the server?
dev-deps aren't installed when installing a package thought npm

Reply:
yes, and If i'm not mistaken slice-simulator is only used for the front-end

Reply:
really?

Reply:
yes